storage/grpc_client.go

```diff
@@ -528,11 +534,21 @@ func (c *grpcStorageClient) DeleteDefaultObjectACL(ctx context.Context, bucket s
 		return err
 	}
 	// Delete the entity and copy other remaining ACL entities.
+	// Note: This API currently does not support entites using project ID.
+	// Use project numbers in ACL entities. Pending b/233617896.
+	// Return error if entity is not found or a project ID is used.
+	invalidEntity := true
 	var acl []ACLRule
 	for _, a := range attrs.DefaultObjectACL {
 		if a.Entity != entity {
 			acl = append(acl, a)
 		}
+		if a.Entity == entity {
+			invalidEntity = false
+		}
+	}
+	if invalidEntity {
+		return fmt.Errorf("entity was not found, got %v. %v", attrs.DefaultObjectACL, errEntityNotSupported)
```

Comment:
Should we use `%w` to wrap the `errEntityNotSupported` error? 
```suggestion
		return fmt.Errorf("entity was not found, got %v. %w", attrs.DefaultObjectACL, errEntityNotSupported)
```

Same thing applies elsewhere if we so choose to wrap the error.
```suggestion
		return fmt.Errorf("entity was not found, got %v. %v", attrs.DefaultObjectACL, errEntityNotSupported)
```

Reply:
Yes indeed! Changed to use `%w` for error values, thanks!